### PR TITLE
Fix #243: Added translations from some translated languages

### LINKED ISSUE #243: Pull finalized localized strings from Transifex

### PR DESCRIPTION
**Pull request by danishjafri88**
Some translations are missing, not yet translated.

## Pull Request Checklist

- [ ] My patch has gone through review and I have addressed review comments
- [ ] My patch has a standard commit message that looks like `Fix #123: This fixes the shattered coffee cup!`

- [ ] I have updated the *Unit Tests* to cover new or changed functionality
- [ ] I have updated the *UI Tests* to cover new or changed functionality
- [ ] I have marked the bug with `[needsuplift]`
- [ ] I have made sure that localizable strings use `NSLocalizableString()`

## Screenshots

If your patch includes user interface changes that you would like to suggest or that you would like UX to look at, please include them here.

## Notes for testing this patch

If useful, please leave notes for QA, explaining what this patch changes and how it can be best tested and verified.

### CODE REVIEW

**Comment by danishjafri88** [comment]
The pull request is huge. Could not do anything from keeping it small. Please give it a test run on multiple languages to verify.

File: Client/Frontend/Strings.swift [hunk 1]
```diff
@@ -17,63 +17,63 @@ private func applicationBundle() -> Bundle {
 }
 
 extension Strings {
-    public static let OKString = NSLocalizedString("OKString", value: "OK", comment: "OK button")
-    public static let ThirdPartySearchFailedTitle = NSLocalizedString("ThirdPartySearchFailedTitle", value: "Failed", comment: "A title explaining that we failed to add a search engine")
+    public static let OKString = NSLocalizedString("OKString", bundle: Bundle.shared, value: "OK", comment: "OK button")
```
Comment (jhreis) [inline Client/Frontend/Strings.swift:20]: Why are we bundling all of these? They aren't technically in the `shared` bundle either.

Comment (danishjafri88) [reply Client/Frontend/Strings.swift:20]: They are in the shared bundle. Check the target membership of the file.

Comment (jhreis) [reply Client/Frontend/Strings.swift:20]: Ah got it, they in the wrong place. Alright, fine by me.

**Review by iccub — CHANGES_REQUESTED** [review]
Edit: last commit fixed the issue for me!

App name is bad on non english regions. I verified it doesn't happen on development branch.

zrzut ekranu 2018-12-4 o 15 33 00

File: BraveShareTo/it.lproj/InfoPlist.strings [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/* Bundle display name */
+"CFBundleDisplayName" = "Apri con Brave";
+
+/* Bundle name */
+"CFBundleName" = "$(PRODUCT_NAME)";
```
Comment (jhreis) [inline BraveShareTo/it.lproj/InfoPlist.strings]: We should remove all these bundle names.

**Review by jhreis — CHANGES_REQUESTED** [review]

File: BraveShareTo/it.lproj/InfoPlist.strings (hunk 2 above)
Comment (danishjafri88) [reply BraveShareTo/it.lproj/InfoPlist.strings]: Ah forgot these, removed

File: BraveShareTo/zh.lproj/InfoPlist.strings [hunk 3]
```diff
@@ -0,0 +1,6 @@
+/* Bundle display name */
+"CFBundleDisplayName" = "在 Brave 中打开";
+
+/* Bundle name */
+"CFBundleName" = "$（PRODUCT_NAME）";
```
Comment (jhreis) [inline BraveShareTo/zh.lproj/InfoPlist.strings]: Remove

File: BraveShared/zh.lproj/InfoPlist.strings [hunk 4]
```diff
@@ -0,0 +1,3 @@
+/* Bundle name */
+"CFBundleName" = "$（PRODUCT_NAME）";
```
Comment (jhreis) [inline BraveShared/zh.lproj/InfoPlist.strings]: Remove

**Review by jhreis — CHANGES_REQUESTED** [review]

**Review by jhreis — APPROVED** [review]

**Review by iccub — APPROVED** [review]

**Pull request merged**